smart_campus/app/views.py

```diff
@@ -262,11 +255,49 @@ def station_edit_page(request, pk):
         'form': form,
         'form_data': form_data,
         'max_imgs': settings.MAX_IMGS_UPLOAD,
+        'images': StationImage.objects.filter(station_id=station.id)
     }
     return render(request, 'app/station_edit.html', context)
 
 
 @login_required
+def set_primary_station_image(request, pk):
+    image = get_object_or_404(StationImage, pk=pk)
+
+    if (not image.station.owner_group == request.user.group and
+            not request.user.is_administrator()):
+        return HttpResponseForbidden()
+
+    station_images = StationImage.objects.filter(station=image.station)
+    for each_img in station_images:
```

Comment:
`each_img` -> `img`
each_img is a weird variable name.

Reply:
I agreed. Fixed it.